v4/source/sharedchannels.yaml

```diff
@@ -1,3 +1,62 @@
+  "/sharedchannels/{team_id}":
+    get:
+      tags:
+        - shared channels
+      summary: Get all shared channels for team.
+      description: |
+        Get all shared channels for a team.
+
+        __Minimum server version__: 5.50
+
+        ##### Permissions
+        Must be authenticated.
+      parameters:
+        - name: team_id
+          in: path
+          description: Team Id
+          required: true
+          schema:
+            type: string
+        - name: page
+          in: query
+          schema:
+            type: integer
+            default: 0
+        - name: per_page
+          in: query
+          schema:
+            type: integer
+            default: 0
+      responses:
+        "200":
+          description: Shared channels fetch successful. Result may be empty.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/SharedChannel"
```

Comment:
It returns array of shared channels right?
```
schema:
                type: array
                items:
                  $ref: "#/components/schemas/SharedChannel"
```

Reply:
Yes

Reply:
Fixed via https://github.com/mattermost/mattermost-api-reference/pull/622/commits/b1162d7ff437a74e815d6cba11b089bb35692af6